docs/sender_adaptors.adoc

```diff
@@ -18,6 +18,51 @@ auto transferred = async::then(t, [] (int i) { return std::to_string(i); });
 // then on s2 it will convert 42 to a string, producing "42"
 ----
 
+WARNING: If the upstream sender _causes_ the downstream scheduler to run, using
+`continue_on` is incorrect: xref:sender_adaptors.adoc#_incite_on[`incite_on`]
+should be used instead.
+
+=== `incite_on`
+
+Found in the header: `async/incite_on.hpp`
+
+`incite_on` is like xref:sender_adaptors.adoc#_continue_on[`continue_on`] but is
+for the use case where running a sender causes a scheduler to be triggered. It
+is intended for use with xref:schedulers.adoc#_trigger_scheduler[`trigger_scheduler`].
+
+[source,cpp]
+----
+auto sndr = async::just([] { send_request(); });
```

Comment:
suggestion: call out here with a comment that the function is being sent to `incite_on`. when I saw this at first I thought of it as `just_result_of` because it was a little strange to see `just` with a lambda.